hosting/digitalocean/files/var/lib/cloud/scripts/per-instance/001_onboot

```diff
@@ -3,7 +3,7 @@
 # go into the app dir
 cd /root
 
-# fetch envoy and docker-compose files
+# fetch nginx and docker-compose files
 wget https://raw.githubusercontent.com/Budibase/budibase/master/hosting/docker-compose.yaml
 wget https://raw.githubusercontent.com/Budibase/budibase/master/hosting/envoy.yaml
```

Comment:
Should we still fetch `envoy.yaml`?

Reply:
Nope - updated all the leftover references to `envoy` now - thanks!